.github/workflows/ui-bff-build.yml

```
@@ -0,0 +1,40 @@
+name: UI - BFF - Build
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    paths-ignore:
+      - 'LICENSE*'
+      - 'DOCKERFILE*'
+      - '**.gitignore'
```

Just an open question: do you want to include only paths of BFF here, or is this the intended semantic?
In general
/lgtm

---

I'm not an expert at writing GHA, so if you feel anything is more correct, just let me know, and I'll fix it!

---

I meant something like:

```suggestion
    paths:
      - 'clients/ui/bff/**'
```

if you want to run it only if BFF is changed, otherwise to run it always the current setup is good for now; that was my question, let me know! 👍

---

Ohh I see. As this is fast to run, I would vote to run in every change on the repo as the BFF will be *really* coupled with model registry rest APIs.

---

sounds good with me 👍  just wanted to double check while looking at this